Providers/nxOMSPerfCounter/MSFT_nxOMSPerfCounterResource.schema.mof

```diff
@@ -8,10 +8,11 @@ class MSFT_nxOMSPerfCounterObject
     [write] string PerformanceCounter[];
 };
 
-[ClassVersion("1.1.0")]
+[ClassVersion("2.0.0")]
```

Comment:
Since I've changed the class definition, I have to change the version number. Bhaskar did the same thing when he updated the parameters of nxOMSAutomationWorker in this commit: https://github.com/Microsoft/PowerShell-DSC-for-Linux/commit/356c91bbf18021c73987ea46a0a0ef87899ec749#diff-a9db1e22016f3c95282253a22658653dR1